cmd/kubeadm/app/cmd/init.go

```diff
@@ -323,6 +323,14 @@ func (i *Init) Run(out io.Writer) error {
 
 	adminKubeConfigPath := filepath.Join(kubeConfigDir, kubeadmconstants.AdminKubeConfigFileName)
 
+	// ReadSentineFile check sentinefile exist
+	if err := kubeadmutil.ReadSentineFile(); err != nil {
```

Comment:
`ReadSentinelFile()` needs to return `string` and `error`.
if `string` != "", then the file was read.
if `error != nil` there was an error.

Reply:
imo we should follow standard Go idioms on Read.